deployments/README.md

```diff
@@ -79,8 +79,49 @@ not strictly required.
 
 ## General Notes
 
-Substrate authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`, `Ferdie`.
-Ethereum authorities are named: `Arthur`, `Bertha`, `Carlos`.
+Rialto authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`.
+Rialto-PoA authorities are named: `Arthur`, `Bertha`, `Carlos`.
+Millau authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`.
+
+Both authorities and following accounts have enough funds on corresponding Substrate chains:
+
+- on Rialto: `Ferdie`, `George`, `Harry`.
+- on Millau: `Ferdie`, `George`, `Harry`.
+
+Names of accounts on Substrate (Rialto and Millau) chains may be prefixed with `//` and used as
+seeds for the `sr25519` keys. This seed may also be used in the signer argument in substrate
+and PoA relays. Example:
+
+```bash
+./substrate-relay rialto-headers-to-millau \
+	--rialto-host rialto-node-alice \
+	--rialto-port 9944 \
+	--millau-host millau-node-alexander \
+	--millau-port 9944 \
+	--rialto-signer //Harry \
+	--prometheus-host=0.0.0.0
+```
+
+Some accounts are used by bridge components. Using these accounts to sign other transactions
+is not recommended, because this may lead to nonces conflict.
+
+Following accounts are used when `poa-rialto` bridge is running:
+
+- Rialto' `Alice` signs relay transactions with new Rialto-PoA headers;
+- Rialto' `Bob` signs relay transactions with Rialto-PoA -> Rialto currency exchange proofs.
+- Rialto-PoA' `Arthur`: signs relay transactions with new Rialto headers;
+- Rialto-PoA' `Bertha`: signs currency exchange transactions.
```

Comment:
I think we should try and avoid calling these `Rialto-PoA`, and stick with either `PoA` or `Ethereum-PoA`

Reply:
While we may (in theory) deploy bridge with any Ethereum-like PoA chain with PoA consensus engine, here we talk about specific chain that has 3 authorities (Arthur, Bertha and Carlos). So I don't think any generic nicknames are suitable here.

Reply:
Sure, but in practice we've only got one instance of the PoA network being used. The deployment files and documentation has been moving away from the `Rialto-PoA` name (although it still lives on a little bit in the code)

Reply:
Well, we have at least two PoA bridges built into Rialto runtime - RialtoPoA and Kovan. We have `RialtoPoAHeaderApi` runtime API. We have `bin/rialto/runtime/src/rialto_poa.rs` file. As I said, I don't think that refering any chain with generic names is ok and if that is done in code, that probably should be fixed.